01-starting-with-data.md

```diff
@@ -284,7 +284,7 @@ class(dat)
 
 ~~~
 
-The output tells us that data currently is a data frame in R.
+The output tells us that dat currently is a data frame in R.
```

Comment:
Here dat is referring to the object `dat`, not the word data. To make that clear, you could add backticks (`) around the word dat. This will format it as inline code.